src/objects/button-group/button-group.scss

```diff
@@ -23,3 +23,12 @@
 .o-button-group--grow > * {
   flex: 1 1 0;
 }
+
+.o-button-group__item {
+  position: relative;
+
+  // TODO This should not live here
+  .c-button {
+    inline-size: 100%;
+  }
+}
```

Comment:
@gerardo-rodriguez Investigating this broke my brain, but the issue seems to be with the inline `padding` on some child elements but not all. The button has padding, [which affects the flexbox width](https://stackoverflow.com/questions/32099624/flexbox-item-widths-are-affected-by-padding), but the `__item` does not, so it takes up less space.

I found [a few different solutions](https://codepen.io/tylersticka/pen/YzjZwbe/23ff47f16f4ed7a5735e01d31f942a47?editors=1100)...

## Option A: Wrap both buttons in `__item`

```html
<div class="o-button-group__item"><!-- copy button, tooltip, etc. --></div>
<div class="o-button-group__item"><!-- draft email link --></div>
```

- **Pros:** Simple
- **Cons:** More markup

(This option feels the most straightforward to me.)

## Option B: Negate inline padding for children of the grow modifier

```css
.o-button-group--grow > * {
  flex: 1 1 0;
  padding-inline: 0;
}
```

- **Pros:** Simple, less markup
- **Cons:** Button content could run into edges at odd sizes

## Option C: Use `grid` instead

```css
.o-button-group--grow {
  display: grid;
  grid-template-columns: repeat(auto-fit, minmax(260px, 1fr));
}

.o-button-group--grow > .o-button-group__item {
  display: grid;
}
```

- **Pros:** Simplifies CSS, less markup
- **Cons:** Relies on an arbitrary "minimum" button width instead of wrapping based on content

## Re: `inline-size` question

No, I don't think we should use `inline-size` here. I think you should mimic whatever layout mechanism is being used by the parent.

```css
.o-button-group__item {
  display: flex;
  position: relative;
}

.o-button-group--grow .o-button-group__item > * {
  flex: 1;
}
```

Reply:
Thanks for digging into this, @tylersticka! I'm going to roll with **Option A**.